src/Components/Loading.tsx

```diff
@@ -1,15 +1,29 @@
+import { PropsWithChildren } from 'react'
 import { connect, currentLanguage } from 'scrivito'
 
-export const Loading = connect(function Loading() {
+export const Loading = (
+  props: object | PropsWithChildren<{ className?: string }> = {},
+) => <LoadingPlaceholder {...props} />
```

Comment:
This looks a bit strange to me. But I have not found a different version, that passes tsc. Good enough for me.